app/views/incomes/payPeriod.scala.html

```diff
@@ -68,7 +70,12 @@
         @form(action=controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.handlePayPeriod, Symbol("data-dynamic-form") -> "true"){
             @{inputRadioGroupStyling(
                 field=payPeriodForm("payPeriod"),
-                radioOptions=Seq(("monthly" -> Messages("tai.payPeriod.monthly")), ("weekly" -> Messages("tai.payPeriod.weekly")),("fortnightly" -> Messages("tai.payPeriod.fortnightly")),("other" -> Messages("tai.payPeriod.other"))),
+                radioOptions=Seq(
+                    PayPeriodForm.MONTHLY -> Messages("tai.payPeriod.monthly"),
+                    PayPeriodForm.WEEKLY -> Messages("tai.payPeriod.weekly"),
+                    PayPeriodForm.FORTNIGHTLY -> Messages("tai.payPeriod.fortnightly"),
+                    PayPeriodForm.OTHER -> Messages("tai.payPeriod.other")
```

Comment:
Since we're in this file, can we use the implicit param instead of the global?

Reply:
😢